app-office/calcurse/calcurse-4.8.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit python-single-r1
+
+DESCRIPTION="A text-based calendar and scheduling application"
+HOMEPAGE="https://calcurse.org/"
+SRC_URI="https://calcurse.org/files/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+IUSE="caldav doc"
+
+RDEPEND="
+	sys-libs/ncurses:0=
+	elibc_musl? ( sys-libs/timezone-data )
```

Comment:
This should not be conditional only for `elibc_musl` if calcurse requires it.

Reply:
This is related to https://github.com/gentoo/gentoo/pull/31008, where I noticed it.